migrations/2020_04_06_180910_create_video_uploads_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateVideoUploadsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        if(! Schema::hasTable('video_uploads')) {
```

Comment:
I'd probably leans towards using `media` rather than a new table.

Reply:
well, if we are to use AWS S3 services for hosting our videos, That means we won't be needing this, tables